exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -19,17 +19,23 @@ public class Game
         LinkedList<string> sportsQuestions = new LinkedList<string>();
         LinkedList<string> rockQuestions = new LinkedList<string>();
 
+        string POP_CATEGORY = "Pop";
+        string SCIENCE_CATEGORY = "Science";
+        string SPORTS_CATEGORY = "Sports";
+
+        int totalQuestions = 50;
+
         int currentPlayer = 0;
         bool isGettingOutOfPenaltyBox;
 
         public Game()
         {
-            for (int i = 0; i < 50; i++)
+            for (int gameIndex = 0; gameIndex < totalQuestions; gameIndex++)
             {
-                popQuestions.AddLast("Pop Question " + i);
-                scienceQuestions.AddLast(("Science Question " + i));
-                sportsQuestions.AddLast(("Sports Question " + i));
-                rockQuestions.AddLast(CreateRockQuestion(i));
+                popQuestions.AddLast("Pop Question " + gameIndex);
+                scienceQuestions.AddLast(("Science Question " + gameIndex));
+                sportsQuestions.AddLast(("Sports Question " + gameIndex));
```

Comment:
Notice how similar the text of the questions in each category are. You've already been provided a method for the generating this for a rock question. Perhaps that method could be generalised?